.changes/improve-wayland-display.md

```diff
@@ -0,0 +1,11 @@
+---
+"tao": patch
+---
+
+Changed the event handling for maximizing to process events sequentially to avoid "Error 71(Protocol error): dispatching to Wayland display".
```

Comment:
```suggestion
On Linux, changed the event handling for maximizing to process events sequentially to avoid "Error 71(Protocol error): dispatching to Wayland display".
```